### PR TITLE
Moved to allowlist naming; added contributing naming rule about potentially offensive terminology.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: cmd/thanos/tools_bucket.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func registerBucketVerify(m map[string]setupFunc, root *kingpin.CmdClause, name
 		Short('r').Default("false").Bool()
 	issuesToVerify := cmd.Flag("issues", fmt.Sprintf("Issues to verify (and optionally repair). Possible values: %v", allIssues())).
 		Short('i').Default(verifier.IndexIssueID, verifier.OverlappedBlocksIssueID).Strings()
-	idWhitelist := cmd.Flag("id-whitelist", "Block IDs to verify (and optionally repair) only. "+
+	idAllowlist := cmd.Flag("id-allowlist", "Block IDs to verify (and optionally repair) only. "+
```
Comment (brancz) [inline cmd/thanos/tools_bucket.go:87]: Is there any additional thing even necessary? Just `id` or `ids` would be even better no?

Comment (bwplotka) [reply cmd/thanos/tools_bucket.go:87]: lol, kind of true (:

**Comment by bwplotka** [comment]
@brancz I was thinking about another example we use (in Prometheus as well) a lot: Blackbox and whitebox monitoring: What about proposing `open/closed monitoring`? :thinking:

**Comment by metalmatze** [comment]
:100:

**Pull request merged**